app/src/main/java/com/myvetpath/myvetpath/CreateSubActivity.java

```diff
@@ -378,7 +404,7 @@ private boolean loadSubmissionData(int status, Submission newSub, SickElement ne
         //Sick element ID - running total
         //Internal ID for sick element table - foreign key from submission table
 
-        if(title_et.getText().toString().isEmpty() || comment_et.getText().toString().isEmpty()){
+        if(title_et.getText().toString().isEmpty() || comment_et.getText().toString().isEmpty() || species.getText().toString().isEmpty()){
```

Comment:
So now the species is a required field. We should set an asterisk next to this field so that the user knows that this is required.

Reply:
Also, I'm not sure if all these fields should be required for a draft.